README.md

```diff
@@ -0,0 +1,53 @@
+# batwatch
+
+_batwatch_ monitors a system's batteries and UPS units for state changes and sends
+notifications via _gpgmailer_.
+
+Also depends on our _confighelper_ library which can be found at
+https://github.com/park-bench/confighelper
+
+batwatch is licensed under the GNU GPLv3. All source code commits prior to the public
+release are also retroactively licensed under the GNU GPLv3.
+
+Bug fixes are welcome!
+
+## System Power States
+
+_batwatch_ looks at the number of batteries installed on a system and each of those
+batteries' charge levels to determine the system's state. If any single battery is
+discharging, the entire system is considered to be discharging.
```

Comment:
You could add something about when the system is considered charging and when the system is considered fully charged. Also you might want to mention that notifications are sent about battery removal or additions.

Reply:
Maybe also describe NO_BATTERY.